src/PrestaShopBundle/Resources/config/services/alias.yml

```diff
@@ -11,3 +11,12 @@ services:
 
     prestashop.twig.extension.dataFormatter:
         alias: prestashop.twig.extension.data_formatter
+
+    hashing:
```

Comment:
Could you add comments about aliases?
Why are they needed?

Reply:
@PierreRambaud there's already [a comment](https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/Resources/config/services/alias.yml#L5) above saying it's for backward compatibility purpose.

Reply:
Ohhh, thanks